### PR TITLE
Add/playground example to ja

### PR DESCRIPTION
**Pull request by sasurau4**
## Overview

Part of #220 

- Diff from original English ver: a6d6fb1
- [English version of v2 playground link](https://www.typescriptlang.org/v2/play?#code/ATD0uB5AHAXBLA9gOwIYBtgGMAWr7IEDmwATgKaq7kAmwAKgMIDMAnMAMqypHnDPYUAZ3LIhAVyHAa40sQCwAKHD8AdAHYA5FJrkAbuXSJoAW1GxVUOEjSZGeAsWAYjAdykBPROOCxEwVzlYciUVLERdAJx4XGxUZGB4EzMaeFRg9A9gIT9oMnFkQmQScgAPaAohIRspVxxRUIh4WGBo2ClUYGRxdExEUmAC3QAzAlpVJUbgAAVSY3JSWCyAQSwscirJ5QgAGXJYbUSTHjGAvjwDZwSMACNxEyiFvlh650X4HIAaK7oXkO23ggclIbkhgCZ4EQcC1kIgWjc+BUNuZEgk-tJ0qhLAAxfrAMqoEzQdD-FSdcJE+DodI2cGoLKwloXPidarFEmAj4WLZLaB8ZboO4mZbTACSACUNtBhHwALzAADeShACFgJIAXNlYHJigBuZWcnIAfk1SsUIBAaDMmpyOqI+vNFtBiBNWrtDotwCReiQkgFQqErttxAA2gBdD3AAC+DpjW10WGpFEEYhat3umv99xFEqlModUwA6s0cMBjAgUBhsA4ikRvl4fFh4gEgqSIOFIuh4ABrZ7RITqrbhVOG2AAITB8vTJiNqlQ72NqmdBYBotTlDoiGGg8USmHOTpHgRywX48nzkF9znp+AADJbxehdegRZl1sVGvfP24iJgLPnFg7TSPAwzDE8yBqlkLzNui95SGy6x3reUyuFSmBUC0qSgeBkFlgkABEwwYEIHj4cAegYOIGzAAAFOQqhEJYzbkESSxusQnxTAADN8AByqC8d88Q0N8riGOg3xEegIgAJQTLuAIwBWtjVvgtYBGheGZL4qC9l0PR9AMQzkKMyC0M4QhTKykIqX4WrGD8ZD7LI1wJMZpnjO+EBKTYVYAKIkmYEHAKs6ybApKizPMixZFQYVSB85FpF+fCWNF6T9N86Llr5mC4GpCgAsR-iuP03a1CWKXAOGZZ8qQGWkLU9TXGsGxsiUgXmEI8l7sILTHEe5Ani+E6IAFLEolOl4zqoIb4fOL74WGs5zc6S0rvu-X0gi2LwI1sCzPovpCFmDxTUKs4LVys7esdp2BrNXERl5VjKVWjAuJZEUQIWzXSJQXbFBpLzAMMBSATUUQxKWA1lgMsOMvipRcs4sBTKQBQIGY3w5ZWeU1k4EjQNKixSCg2lNr0TidGDyAQygUwgYkLRlFy3UMN+TYJBQxJUHwHZ8HUTzAA2ATeOgvz1akb29B4KGttkiBmC8Thdr2mpMzRtNYDJoPgzRMkvbiAwEkSHL1BQhzNrjKmU5gdnonbNxUN2oNzCYVkJDmTkAI5UTkO69SOdsAKrQDQ6TkAAsvsqAR9wwDyjRytx5imrxB4uuygAfDM7sfPRlSIOgBjJ7H8eoDJupgBA2K6fyYpxL0Q59YM4eR6dMfcBXicWR4dO0SnFfp8gHjfE7Luugbie53oiDwDQWe52aICoK4+AtKH7fBF3qfcGX3eYlXWwgBPWDdrOBuxiuKgAJreHE3MbuC-Qsjc3gtDbVb5Y4QMEFVzANDABuEYEg0oA6NCmNCWA0ABzgF0HoEBiAiDdQhFgOYQgtwWApKAXkGx0HwDgKAeIsJwbEAALR4KEAQuA5DmDkPUKAIAA)

### CODE REVIEW

File: packages/playground-examples/copy/ja/3-7/Syntax and Messaging/Flattened Error Reporting.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
+//// { compiler: {  }, order: 3 }
+
+// TypeSciprtのエラーメッセージはときどき、少しだけ冗長になってしまうことがあります。
```
Comment (orta) [inline packages/playground-examples/copy/ja/3-7/Syntax and Messaging/Flattened Error Reporting.ts:3]:
```suggestion
// TypeScriptのエラーメッセージはときどき、少しだけ冗長になってしまうことがあります。
```

Comment (sasurau4) [reply packages/playground-examples/copy/ja/3-7/Syntax and Messaging/Flattened Error Reporting.ts]: crazy typo :eyes:

**Comment by orta** [comment]
This ready to merge?

**Comment by sasurau4** [comment]
@orta OK 👍

**Pull request merged**